fuzz/fuzz_targets/simple_model.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2021-2022 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or MIT.
+
+//! Model check without reference counting:
+//! Checks that a sequence of operations and restarts behaves the same as an in-memory collection.
+
+#![no_main]
+use arbitrary::Arbitrary;
+use libfuzzer_sys::fuzz_target;
+use std::collections::btree_map::BTreeMap;
+use tempfile::tempdir;
+
+#[derive(Arbitrary, Debug)]
+enum Action {
+	Transaction(Vec<(u8, Option<u8>)>),
+	Restart,
```

Comment:
I never did use Arbitrary, looks nice.
I could probably rewrite some code with it.